src/ducks/apps/components/ApplicationPage/Details.jsx

```diff
@@ -40,20 +40,21 @@ export class Details extends Component {
     this.props.history.push(`/${parent}/${app.slug}/permissions`)
   }
 
-  toggleChannels() {
+  async toggleChannels() {
     const { app } = this.props
-    if (!getChannel(app.source))
-      console.warn("This application don't use the registry")
+    // only for installed apps and apps from the registry
+    if (!getChannel(app.source) || !app.installed) return
```

Comment:
Nitpick:
```
if (!(app.installed && getChannel(app.source))
```
is clearer IMHO and we save a call to `getChannel` it the app is not installed